src/notation/qml/MuseScore/NotationScene/internal/ElementPopup.qml

```diff
@@ -0,0 +1,34 @@
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+import QtQuick.Controls 2.15
+import QtQuick.Controls 1.3
+
+import MuseScore.NotationScene 1.0
+import MuseScore.UiComponents 1.0
+import MuseScore.Ui 1.0
+import MuseScore.Inspector 1.0
+
+
+StyledPopup {
+    id: root
+
+    property var model;
+    property string type;
+
+    closePolicy: Popup.CloseOnPressOutside
```

Comment:
Is there any reason not to use `Popup.CloseOnEscape | Popup.CloseOnPressOutside` instead of `Popup.CloseOnPressOutside`?